jwt/service/jwt_token_service.go

```diff
@@ -161,18 +163,18 @@ func (ts *JWTokenService) ValidateTokenString(tstr string, v jwtValidator.Valida
 }
 
 // NewAccessToken creates new access token for user.
-func (ts *JWTokenService) NewAccessToken(u model.User, scopes []string, app model.AppData, requireTFA bool) (ijwt.Token, error) {
+func (ts *JWTokenService) NewAccessToken(u *proto.User, scopes []string, app model.AppData, requireTFA bool) (ijwt.Token, error) {
```

Comment:
I am not sure this is a good idea to connect protobug to this app layer, we need to keep protobuf isolated in the plugin layer.